### ISSUE TITLE
Portfolio Management

### ISSUE THREAD

**Issue by neomodern** [drpout/boilr#272]
This entails a new column which shows user configurable values for how many coins they hold of a cryptocurrency. This column can copy Blockfolio's UI and go between the pair/exchange and upper/lower limits columns.

For example a BTC/USD pair can be clicked and in the alarm settings the user can type in how many BTC they currently hold. It would then display both "1 BTC" and "3850 USD" in that alarm row.

**Comment by dllud** [comment]
Thanks for the suggestion. This is something we have considered in the past. At the time the main reasons against it were:

1. Lack of space on the UI. We were targeting 480p screens. I think we may now upgrade our min specs to 720p and redesign the alarm list. This may handle us enough space for the portfolio info and for #258.

2. Mixing too many features in the same app, thus departing from the UNIX philosophy of "do one thing and do it well". Right now Boilr is just a price alarm app. Besides portfolio tracking, we can conceive many many more related and useful features, such as real portfolio management (charts, history, etc.), trading operations (buy/sell, exchange) and whatnot. If we keep adding features Boilr will become a big monolith for everything cryptotrade and cryptoinvestment related. That's something we cannot maintain with our limited resources.

**Comment by neomodern** [comment]
Fair enough, I can see the necessity of keeping Boilr lean and focused. 

Though there is something to be said for becoming a one-stop shop - people love the power of Blockfolio but many such as myself have realised it's a privacy risk. A larger featureset may also encourage more collaborators and donators. 

But at any rate simple portfolio tracking should be sufficient for the basic usecase of "how much have I currently got of X cryptocurrency" to make the price alerts more applicable.
